src/calendar/Calendar.ts

```diff
@@ -260,7 +234,7 @@ export default class Calendar extends CalendarBase<CalendarProperties> {
 			month,
 			year
 		} = this._getMonthYear();
-		const { theme = {} } = this.properties;
+		const { theme = {}, customDateCell } = this.properties;
```

Comment:
should this be updated to `CustomDateCell` to be consistent with the other case changes?